### DISCUSSION TITLE
Allow card view by default in default source settings in index pages

### DISCUSSION THREAD

**Discussion by MoritzLost** [craftcms/cms#15216] [Ideas]
All element indexes can now be viewed in card view, which is very useful for a lot of sections. But all editors have to manually select the card view, it's not possible to set the card view as a default for all users.

When editing the default settings for sources (both default and custom sources) in an element index, it would be cool to have an option to change the default view mode from list/structure view to card view, next to the default sort:

![Screenshot 2024-06-17 at 14 57 31](https://github.com/craftcms/cms/assets/10146880/9245b671-4409-4164-a64f-d42524216d1a)

**Comment by MangoMarcus** [comment]
+1 for this 👍
